server/src/modules/mixin/rebalance/rebalance.service.ts

```diff
@@ -221,12 +221,12 @@ export class RebalanceService {
     const minBalance = new BigNumber(minimum_balance);
 
     // Determine if rebalance is necessary
-    if (balanceBN.isLessThanOrEqualTo(minBalance)) {
+    if (!balanceBN.isLessThanOrEqualTo(minBalance)) {
       return;
     }
-
+    console.warn(balanceBN.minus(minBalance).toString());
     // Calculate the amount needed to rebalance
-    const amountToRebalance = balanceBN.minus(minBalance);
+    const amountToRebalance = minBalance.minus(balanceBN);
```

Comment:
As we have `(!balanceBN.isLessThanOrEqualTo(minBalance)) {return}` above.
The `balanceBN` here is greater than `minBalance`. 
So the `amountToRebalance` would always be smaller than 0.
And it will never trigger rebalance?

Also we can replace `!balanceBN.isLessThanOrEqualTo(minBalance)` with `isGreaterThan`